<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>667</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame#mainFrame{
	
	border-image: url(:/MainFrame/mainFrame.png);

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="contentFrame">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>111</y>
         <width>59</width>
         <height>18</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>600</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame#contentFrame{

border:None;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>50</number>
        </property>
       </layout>
      </widget>
      <widget class="QFrame" name="windgedFrame">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>496</y>
         <width>919</width>
         <height>120</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>120</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{

border:None;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>2</width>
            <height>2</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="MRNumber">
           <property name="geometry">
            <rect>
             <x>275</x>
             <y>9</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tlwg Typist</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: white;
	

}</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="parentNumberText">
           <property name="geometry">
            <rect>
             <x>3</x>
             <y>50</y>
             <width>50</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:Black;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;NP&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QSlider" name="numberParentSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>53</x>
             <y>45</y>
             <width>201</width>
             <height>29</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    border: 3px solid;
    height: 4px;
    margin: 0px;
	;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 188, 191, 255), stop:1 rgba(255, 255, 255, 255));
    }
QSlider::handle:horizontal {
    background-color: yellow;
    border: 1px solid;
    height: 4px;
    width: 12px;
    margin: -5px 0px;
    }
QSlider::groove::disabled{
	background-color: Black;
	border: 1px solid;
	border-color: Black;
}
QSlider::handle::disabled{
	background-color: Black;
	border: 1px solid;
	border-color: Black;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QSlider" name="mutationRateSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>56</x>
             <y>9</y>
             <width>206</width>
             <height>29</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QSlider::groove:horizontal {
    border: 3px solid;
    height: 4px;
    margin: 0px;
	;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(127, 191, 0, 255), stop:1 rgba(255, 255, 255, 255));
    }
QSlider::handle:horizontal {
    background-color: yellow;
    border: 1px solid;
    height: 4px;
    width: 12px;
    margin: -5px 0px;
    }
QSlider::groove::disabled{
	background-color: Black;
	border: 1px solid;
	border-color: Black;
}
QSlider::handle::disabled{
	background-color: Black;
	border: 1px solid;
	border-color: Black;
}</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="mutationRateText">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>40</width>
             <height>20</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:Black;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;MR&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
          <widget class="QLabel" name="NPNumber">
           <property name="geometry">
            <rect>
             <x>274</x>
             <y>50</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Tlwg Typist</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel{
	color: white;
	

}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="populationNumberInput">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>310</x>
             <y>50</y>
             <width>140</width>
             <height>25</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>KacstDecorative</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
	background-color:white;
	color: Black;
}

QLineEdit:focus{
	 select All;
}

QLineEdit::Disabled{
	Background-color: Gray;
}</string>
           </property>
           <property name="text">
            <string>Population No</string>
           </property>
           <property name="cursorPosition">
            <number>13</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="startBTN">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>770</x>
             <y>50</y>
             <width>75</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	background-color:Green;
	color:White;
}

QPushButton::disabled{
	background-color: Gray;
}</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QPushButton" name="addPointBTN">
           <property name="geometry">
            <rect>
             <x>650</x>
             <y>50</y>
             <width>101</width>
             <height>27</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:Green;
color:White</string>
           </property>
           <property name="text">
            <string>Add Point</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Background/add_data.png</normaloff>../Background/add_data.png</iconset>
           </property>
          </widget>
          <widget class="QComboBox" name="typeSelection">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>470</x>
             <y>50</y>
             <width>151</width>
             <height>27</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox::Disabled{
	background-color: Gray;
}</string>
           </property>
           <item>
            <property name="text">
             <string>Ranks</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Roulette</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Random</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="selectionTypeText">
           <property name="geometry">
            <rect>
             <x>470</x>
             <y>30</y>
             <width>121</width>
             <height>17</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">color:White;
background-color:None</string>
           </property>
           <property name="text">
            <string>Selection Type</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="screenFrame">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>961</width>
         <height>441</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Background/MainFrame.qrc"/>
 </resources>
 <connections/>
</ui>
